src/ucp/tag/rndv.c

```diff
@@ -730,7 +731,7 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_put_zcopy, (self),
                                    status);
     if (sreq->send.state.dt.offset == sreq->send.length) {
         if (sreq->send.state.uct_comp.count == 0) {
-            ucp_rndv_send_atp(sreq, sreq->send.rndv_put.remote_request);
+            sreq->send.state.uct_comp.func(&sreq->send.state.uct_comp, status);
```

Comment:
what is the reason to change this?

Reply:
In cuda pipeline fragmentation, put_zopy will be called with frag_req(not original sreq) object.   ucp_rndv_send_atp() will do ucp_request_complete_send on the sreq, which is incorrect.

In uct_comp.func (ucp_rndv_frag_put_completion), does cleanup(release frag_req) and call send_atp on original sreq.